cachito/workers/pkg_managers/rubygems.py

```diff
@@ -378,6 +383,22 @@ def resolve_rubygems(package_root, request):
     }
 
 
+def unpack_git_dependency(dep):
```

Comment:
Should we add a separate unit test for this method? Coverage dropped a little bit.

Reply:
Yes, it's added